BMC - implement OAuth2 and add bmc-itsm-add-attachment command (#43166)
demisto/content: Packs/BmcITSM/ReleaseNotes/1_0_46.md

#### Integrations

##### BMC Helix ITSM

- Added support for **Client ID** parameter that required when 'use oauth' is selected.
- Added support for **Authorization Code** parameter that obtained by running the **bmc-itsm-generate-login-url** command and following the url. required when 'use oauth' is selected.
- Added support for **Redirect URI** parameter that required when 'use oauth' is selected.

> - Added support for **OpenID Issuer URL** parameter that the rsso server url. for example https://<rsso_server_name>:8443/rsso. required when 'use oauth' is selected.

Reviewer: ```suggestion
- Added support for the *OpenID Issuer URL* parameter to set the RSSO server URL. Required when 'Use Oauth' is selected.
```

#### Integrations

##### BMC Helix ITSM

- Updated the Docker image to: *demisto/python3:3.12.13.7444307*.
- Added support for the *Client ID* parameter. Required when 'Use Oauth' is selected.

> - Added support for the *Authorization Code* parameter, which is obtained by running the **bmc-itsm-generate-login-url** command and following the URL. Required when 'Use Oauth' is selected.

Reviewer: ```suggestion
- Added support for the *Authorization Code* parameter, which is obtained by running the ***bmc-itsm-generate-login-url*** command and following the URL. Required when 'Use Oauth' is selected.
```
Sorry, I just realized I missed that small formatting change
